web/cypress/e2e/onboarding-as-foreign-business.spec.ts

```diff
@@ -158,4 +158,49 @@ describe("Onboarding for all industries when out of state nexus business [featur
       cy.get('[data-testid="header-link-to-profile"]');
     });
   });
+
+  describe("Onboarding for all industries when out of state remote worker business [feature] [all] [group4]", () => {
```

Comment:
pebble - the title needs to be updated to reflect the test

Reply:
we are not testing industries

Reply:
updated